extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzRuntimeConfig.java

```diff
@@ -29,8 +31,23 @@
      * By default, the scheduler is not started unless a {@link io.quarkus.scheduler.Scheduled} business method is found.
      * If set to true the scheduler will be started even if no scheduled business methods are found. This is necessary for
      * "pure" programmatic scheduling.
+     *
+     * @deprecated use quarkus.quartz.start-mode=forced instead.
      */
+    @Deprecated
     @ConfigItem
-    public boolean forceStart;
+    public Optional<Boolean> forceStart;
 
+    /**
+     * Scheduler can be started in different modes: normal, forced or halted
+     * By default, the scheduler is not started unless a {@link io.quarkus.scheduler.Scheduled} business method
+     * is found. (normal mode)
+     * If set to "forced", scheduler will be started even if no scheduled business methods are found.
+     * This is necessary for "pure" programmatic scheduling.
+     * Additionally, setting it to "halted" will behave just like forced mode but the scheduler will be not start
+     * triggering jobs until an explicit start is called from the main scheduler.
+     * This is useful to programmatically register listeners before scheduler starts performing some work.
+     */
+    @ConfigItem
+    public Optional<String> startMode;
```

Comment:
Let's re-use the enum declared in https://github.com/quarkusio/quarkus/pull/14043/files#diff-4996139067b2db98f1af6397a79d30707d6fa1a376ba3afa453c95bcaa012582R6